api/handlers.go

```diff
@@ -541,7 +541,11 @@ func (api *API) userRecoverGET(w http.ResponseWriter, req *http.Request, _ httpr
 		if errSend != nil {
 			api.staticLogger.Warningln(errors.AddContext(err, "failed to send an email"))
 		}
-		api.WriteError(w, errors.AddContext(err, "no user with this email"), http.StatusBadRequest)
+		// We don't want to give a potential attacker information about the
+		// emails in our database, so we will respond that we've sent the email.
+		// If they used the wrong email, they will get an email that indicates
```

Comment:
Shouldn't this be "If they used the correct email address"?

Reply:
I might have to reword that but this is exactly what I meant:
- if they used the correct email, it's obvious that they would get an email, that's the basic idea of the operation
- if they used the wrong email, i.e. another email of theirs that they didn't use to sign up for Skynet, they would get an email indicating that someone tried to use that email in order to recover an account. And then they would hopefully go "Oh! Silly me, I tried using `blabla@example.com` and I signed up using `lalala@example.com`!" and they will request another recovery with the right email address.